parseArtifacts.js

```diff
@@ -0,0 +1,53 @@
+const fs = require("fs");
+const path = require("path");
+const mkdirp = require("mkdirp");
+
+const parseArtifact = (inFile, outDir, outFile, type = "abi") => {
```

Comment:
For dev purposes, currently using Truffle to compile contracts and then using this script to parse out the ABI and bytecode for contracts. Contract ABI and bytecode is then used to generate Go bindings for the Go client whenever a contract is updated